buildSrc/version.properties

```diff
@@ -27,6 +27,7 @@ guava             = 31.1-jre
 jna               = 5.5.0
 
 netty             = 4.1.91.Final
+nettytcnative     = 2.0.54.Final
```

Comment:
Where is this used?

Reply:
I have removed `nettytcnative` from here for now, will take up once I start work on OpenSSL migration.